models/staging/hubspot_staging/stg_rag_hubspot__deal.sql

```diff
@@ -36,7 +36,7 @@ fields as (
 final as (
 
     select
-        deal_name,
+        deal_name as title,
```

Comment:
This needs to be documented as a breaking change in the CHANGELOG

Reply:
added